hadoop-hdds/server-scm/src/test/java/org/apache/hadoop/hdds/scm/container/placement/algorithms/TestContainerPlacementFactory.java

```diff
@@ -163,10 +163,13 @@ public void testRackAwarePolicy() throws IOException {
   }
 
   @Test
-  public void testDefaultPolicy() throws IOException {
+  public void testRackAwareContainerPolicy() throws IOException {
+    conf.set(ScmConfigKeys.OZONE_SCM_CONTAINER_PLACEMENT_IMPL_KEY,
+            SCMContainerPlacementRackAware.class.getName());
     PlacementPolicy policy = ContainerPlacementPolicyFactory
         .getPolicy(conf, null, null, true, null);
-    Assertions.assertSame(SCMContainerPlacementRandom.class, policy.getClass());
+    Assertions.assertSame(SCMContainerPlacementRackAware.class,
+            policy.getClass());
   }
```

Comment:
I would have preferred keeping the intention of this test: testing default setting.  Sorry for not noticing this earlier.